tests/e2e/_test_sem_covid_config.py

```diff
@@ -59,8 +61,14 @@ def test_vault_config():
     assert config.VAULT_ADDR is not None
 
 
-def test_flask_config():
-    assert FlaskConfig.FLASK_SECRET_KEY is not None
-
+def test_ml_experiments_config():
```

Comment:
this test, could perhaps run all the time (remove the _underscore); We need all these variables